### PR TITLE
Examples formatting

### PR DESCRIPTION
**Pull request by autra**
First pass at cleaning the examples (there will be others I'm sure).

They will be part of our vitrine for our users, let's make them shiny and polished!

@Jeremy-Gaillard please have a look at the 3dtiles examples. I have integrated the debug functions into Debug.js. 

About Debug.js, I think it should be splitted soon...

@iTowns/reviewers please r?

### CODE REVIEW

**Review by peppsac — DISMISSED** [review]
Looks good, r+

**Comment by Jeremy-Gaillard** [comment]
Thanks for the work @autra.

Just two remarks:
- Please set one of the layer visible by default (3d-tiles-discrete-lod)
- The text from the debug bbox of the request volume tileset stays displayed even after hiding the debug layer

**Comment by autra** [comment]
@Jeremy-Gaillard updated. I've also updated the starting position

@peppsac please r? again, especially the last commit

Thanks!

**Comment by autra** [comment]
@peppsac please r?

File: examples/multiglobe.html [hunk 1]
```diff
@@ -8,21 +8,29 @@
     <head>
         <title>Globe</title>
```
Comment (peppsac) [inline examples/multiglobe.html]: You might want to change the title as well

File: examples/planar.html [hunk 2]
```diff
@@ -3,109 +3,33 @@
         <title>Globe</title>
```
Comment (peppsac) [inline examples/planar.html]: Change title ?

File: examples/postprocessing.html [hunk 3]
```diff
@@ -3,9 +3,16 @@
     <head>
         <title>Globe</title>
```
Comment (peppsac) [inline examples/postprocessing.html]: Title...

**Review by peppsac — COMMENTED** [review]
+1
You only need to fix a few `<title>` and we'll be able to merge this PR.

**Review by peppsac — APPROVED** [review]

**Pull request merged**